### DISCUSSION TITLE
How to expand navigation only for a single page/section?

### DISCUSSION THREAD

**Discussion by TRaSH-** [squidfunk/mkdocs-material#2185] [Q&A] [answered]
Is it possible to have the navigation expand for 1 page only and not the others ?
At the moment i've one page that looks like this because i removed the 0.2 folder
![image](https://user-images.githubusercontent.com/6155095/103675831-af46e800-4f80-11eb-9c81-47e3f19e9d03.png)

And it would be nice if i only can auto expand it for this page.

![image](https://user-images.githubusercontent.com/6155095/103675990-db626900-4f80-11eb-9149-9c883f35357e.png)

If i would remove the v3 folder it we will break all direct links to it (from several external links),
and when Radarr decides to start with Radarr v4 it would mean I need to move it again.

**Comment by squidfunk** [comment]
I'm sorry, I don't really understand the question. Did you try [navigation expansion](https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/#navigation-expansion)?

**Comment by TRaSH-** [comment]
Yes i tried it but then it expand it for all page and not only for 1 page.

**Reply by squidfunk** [reply]
I see. Unfortunately, it's not possible to activate this for a single page (or respectively section).

**Reply by squidfunk** [reply]
I added my comment as an answer again, as the answer was buried in the comment. It's easier for users to find it in the future now.

**Comment by TRaSH-** [comment]
okay thnx,
i was also testing  navigation expansion local but this didn't work running local.

**Reply by squidfunk** [reply]
BTW, if you want to provide docs for different versions, you can do so by [adding a version selector](https://squidfunk.github.io/mkdocs-material/setup/setting-up-versioning/).

**Comment by squidfunk** [comment, marked as answer]
Unfortunately, it's not possible to activate this for a single page (or respectively section).
